### ISSUE TITLE
CMake: "No project() command is present"

### ISSUE THREAD

**Issue by rotu** [ros2/tinydir_vendor#18]
```
--- stderr: tinydir_vendor                                                                                                                      
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

---
Finished <<< tinydir_vendor [4.52s]
```

**Comment by dirk-thomas** [comment]
What ROS distro are you seeing this problem with?

**Comment by rotu** [comment]
I think this was Eloquent, and it looks like I already fixed this in #9. I can't figure out how I encountered it again, but probably user error.
